composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
-        "behat/behat": "^3.0",
+        "behat/behat": "^3.9",
```

Comment:
See https://github.com/nicwortel/behat-unused-step-definitions-extension/actions/runs/9601338720/job/26479666275#step:5:193
Using dataProvider in tests triggered this,  but we have to raise the constraint on behat because 3.0 is not compatible with PHP 8.1 (for symfony dependencies), only 3.9 and above is.

Reply:
@vever001, I think we can do it in the PR. As this package is indicating PHP 8.1 as minimal version (https://github.com/nicwortel/behat-unused-step-definitions-extension/blob/master/composer.json#L7) and Behat <3.9, because of its dependencies, doesn't work with PHP 8.1, I think we fix the Behat constraint in composer.json

Reply:
Exactly, that is what I did.